### DISCUSSION TITLE
Poll-Schumann Model Aircraft Parameters

### DISCUSSION THREAD

**Discussion by meessn** [contrailcirrus/pycontrails#278] [Q&A]
Hi,

I have been simulating A320neo flights powered by a PW1127G with the PS model. The PS Model aircraft parameters CSV contains info on the A320neo, such as the maximum and idle fuel flow at sea level static conditions and the rated thrust (so from ICAO). I see that these deviate from the ICAO databank for the PW1127G engine. I checked that the PW1127G is the standard engine used in the A20N model. Also other values such as eta_1, eta_2, nominal bpr are different from the paper published by Poll and Schumann (Part2, 2021). I was wondering how these parameters are determined?

Thank you very much!

**Comment by zebengberg** [comment]
Tagging @roger-teoh

**Comment by roger-teoh** [comment]
Hi @meessn , thank you for this question. The Airbus A320neo can be powered by two different engine options, such as the CFM LEAP-1A or the PW engine. While we expect some difference in the fuel consumption between the different engine options, the PS model coefficients does not differentiate between the different engine options. Therefore, it is likely that the coefficients were calibrated using the CFM LEAP engine. 

We note that the PS model coefficients in the static files are continuously updated by Ian Poll as more data becomes available. Based on my understanding, the PS model coefficients were derived based on the Flight Crew Operating Manual (FCOM) data. The latest update to the model coefficients occurred around May-2024, so any coefficients listed in the paper "Poll and Schumann (Part2, 2021)" has been superseded with more up to date values. 

Hope this helps.
